skrypty/ui/talk_window/talk_history.lua

```diff
@@ -0,0 +1,23 @@
+scripts.ui.talk_history = scripts.ui.talk_history or {
+    content = {},
+    size = 20
+}
+
+function scripts.ui.talk_history:add(message)
```

Comment:
Moznaby to dodac jako abstract util zeby miec mozliwosc struktur kolejkowych z okreslonym length

Reply:
Myślałem o tym, wrócę do tematu na pewno.